### DISCUSSION TITLE
Jump to a specific Heading that starts with a Number?

### DISCUSSION THREAD

**Discussion by Wikifanboy** [requarks/wiki#6224] [Help / Questions] [answered]
Hello,

im trying to jump from Page 1 to Page 2 to a specific Heading that starts with a Number.

Example:
Page Rules: Security (click on Security)
jump to
Page Security: 6 Customer Security (The heading i want to jump to)

The problem im facing is the number 6 in the beginning. Without the number it works just fine with `#customer-security`
But as soon as i try to start with a number, which is needed (there is no alternative for me) it doesn’t work anymore.

I tried the following commands where none of them works:
`#6-customer-security`
`#6customer-security`
`#6_customer-security`

Is there any way to get this working in WYSIWYG or any other solution (HTLM code etc.) or is it just the fact that numbers are not allowed in the beginning of a Header?

I would appreciate any solution to this problem.

Kind regarts

**Comment by fnafox** [comment, marked as answer]
If you click on the paragraph indicator on the right of the heading you will see the exact link to the section in question, if you copy the link, it will lead you to that section of the page.

![image](https://user-images.githubusercontent.com/103725434/222728873-2ad31e08-7d2e-4ac1-9a9a-a6d8a1357eb0.png)

**Reply by Wikifanboy** [reply]
> If you click on the paragraph indicator on the right of the heading you will see the exact link to the section in question, if you copy the link, it will lead you to that section of the page.
> 
> ![image](https://user-images.githubusercontent.com/103725434/222728873-2ad31e08-7d2e-4ac1-9a9a-a6d8a1357eb0.png)

Thank you so much, didn’t know about this feature, thats exactly what fixed it.

So the Syntax is `#h-6-customer-security`

You saved my Day :)
